### ISSUE TITLE
19197 - Additional Recommendations for New Servers - CentOS 6.6

### ISSUE THREAD

**Issue by gnakan** [gnakan/gd-cloud-docs#102]

**Comment by brian-r-miller** [comment]
Following your instructions yields the following result:

[randomcode@50Cent ~]$ rpm -q iptables
iptables-1.4.7-16.el6.x86_64
[randomcode@50Cent ~]$ sudo service iptables status
iptables: Firewall is not running.
[randomcode@50Cent ~]$ sudo service iptables start
iptables: No config file.                                  [WARNING]

Article assumes firewall is running

**Comment by kharise** [comment]
Yup - so centos 6 comes with iptables, it just needs to be enabled, and by default its config file is empy, but the article defines  all the rules (in  Create firewall rules)
